rollup.config.js

```diff
@@ -0,0 +1,12 @@
+import buble from 'rollup-plugin-buble'
+
+export default {
+  entry: 'src/index.js',
+  dest: 'dist/multicast.js',
+  format: 'umd',
+  moduleName: 'most',
```

Comment:
i think it should be `mostMulticast` here as well, at least that how we have it in the `mostjs/create` since its the name of the global export?